common_benchmark_suite/openxla/benchmark/models/jax/generate_model_artifacts.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import argparse
+import importlib
+import jax
+import jax.numpy as jnp
+import os
+import pathlib
+import re
+import multiprocessing
+import shutil
+import subprocess
+import sys
+import tarfile
+
+from typing import Any, Optional
+
+# Add openxla dir to the search path.
+sys.path.insert(0, str(pathlib.Path(__file__).parents[4]))
+
+from openxla.benchmark import def_types
+from openxla.benchmark.comparative_suite.jax import model_definitions
+from openxla.benchmark.models.jax import model_interfaces
+
+HLO_FILENAME_REGEX = r".*jit_forward.before_optimizations.txt"
+HLO_STATIC_FILENAME = "xla_hlo_before_optimizations.txt"
+
+
+def _generate_and_save_inputs(model_obj: Any,
+                              model_dir: str) -> tuple[Any, ...]:
+  raw_inputs = model_obj.generate_default_inputs()
```

Comment:
We can have different raw inputs and they can be fetched from GCS. Could you file an issue and add TODO here?

Reply:
Done